SDKs/ServiceBus/data-plane/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <PropertyGroup>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <LangVersion>7.1</LangVersion>
```

Comment:
I recently set LangVersion=latest in common props so unless there is a strong need to pin the version I'd exclude this. As for TreatWarningsAsErrors I'm fine with it being here for now but I plan to turn it on globally once I can get all the warnings fixed.

Reply:
These were carried over from the original project file.  I hadn't thought to revisit, but in doing so, the build and test run comes back clean without the version pinned;  taking it out.